src/schemas/json/hammerkit.json

```diff
@@ -0,0 +1,422 @@
+          "type": "string"
+        },
+        "image": {
+          "title": "task image",
+          "description": "Container image for the current task to run commands in",
+          "type": "string"
+        },
+        "ports": {
+          "description": "A list of ports that get exposed on the host from the container <localPort>:<targetPort>",
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "description": "<localPort>:<targetPort>",
+            "type": "string",
+            "pattern": "\\d+:\\d+"
+          }
+        },
+        "mounts": {
+          "description": "File paths that get mounted into the container\nhttps://no0dles.gitbook.io/hammerkit/task/container#adding-mounts",
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "type": "string",
+            "pattern": "\\d+:\\d+"
+          }
+        }
+      }
+    },
+    "taskCommand": {
+      "anyOf": [
+        {
+          "title": "command",
+          "description": "Command to be executed for the current task",
+          "type": "string"
+        },
+        {
+          "title": "task command with path extension",
+          "description": "Command to be executed in sub directory for the current task",
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["cmd"],
+          "properties": {
+            "cmd": {
+              "title": "command",
+              "description": "Command to be executed for the task",
+              "type": "string"
+            },
+            "path": {
+              "title": "command path",
+              "description": "Path below build file path",
+              "$ref": "https://json.schemastore.org/base.json#/definitions/path"
+            }
+          }
+        }
+      ]
+    },
+    "taskDescription": {
+      "title": "task description",
+      "description": "Description for the current task",
+      "type": "string"
+    },
+    "taskExtend": {
+      "title": "task extension",
+      "description": "Extension task as a base template\nhttps://no0dles.gitbook.io/hammerkit/task/extending",
+      "type": "string"
+    },
+    "taskCache": {
+      "title": "task cache",
+      "description": "compare type for detecting changes in source files\nhttps://no0dles.gitbook.io/hammerkit/task/caching",
+      "type": "string",
+      "enum": ["checksum", "modify-date", "none"]
+    },
+    "taskContinuous": {
+      "title": "task continuous",
+      "description": "The task runs continuous and does not end without cancellation\nhttps://no0dles.gitbook.io/hammerkit/task/watching#continuous-tasks",
+      "type": "boolean"
+    },
+    "taskLabels": {
+      "title": "label options",
+      "description": "A map of label values for the current task\nhttps://no0dles.gitbook.io/hammerkit/labels",
+      "type": "object",
+      "additionalProperties": { "type": "string" }
```

Comment:
I am not sure about this: try to check whether adding `description` for such properties will be shown in VS Code (if u use it).

Reply:
not a regular user of VS Code. could not set it up with the `json.schemas` settings, so it works locally to test. 

Tested with Webstorm where just the description is used on objects. Sadly neither `"additionalProperties": { "type": "string" }` nor `patternProperties` has any effect on validation.

Reply:
`description` works for `additionalProperties` as I've expected: it's being shown when hovering such props:

![image](https://user-images.githubusercontent.com/42812113/210301424-258ce382-a361-4476-b32e-97c40eedbae0.png)

```json
{
  "$schema": "http://json-schema.org/draft-07/schema",
  "type": "object",
  "properties": {
    "first": {
      "description": "A first property",
      "type": "string"
    }
  },
  "additionalProperties": {
    "description": "An additional property"
  }
}
```
This is a schema being tested.

Reply:
thanks, thats helpful 👍